onnxruntime/core/providers/webnn/builders/impl/lstm_op_builder.cc

```diff
@@ -23,9 +23,9 @@ class LstmOpBuilder : public BaseOpBuilder {
 
   // Operator support related.
  private:
-  bool IsOpSupportedImpl(const InitializedTensorSet& initializers, const Node& node,
+  bool IsOpSupportedImpl(const GraphViewer&, const Node& node,
```

Comment:
Nit: `graph_viewer` is used.
```suggestion
  bool IsOpSupportedImpl(const GraphViewer& graph_viewer, const Node& node,
```